src/lib.rs

```diff
@@ -70,9 +70,18 @@
 
 #![deny(warnings, missing_docs, missing_debug_implementations)]
 #![doc(html_root_url = "https://docs.rs/bytes/0.4.12")]
+#![no_std]
 
+// TODO: remove this when alloc stabilization lands in the 'nightly' channel
+#![cfg_attr(all(feature = "alloc", not(feature = "std")), feature(alloc))]
```

Comment:
Looks like `alloc` stabilization isn't quite in the latest nightly on Travis CI, but as soon as it is this can be removed.

Reply:
Update: removed now that this has been stabilized in the nightly channel